contract/views/contract.xml

```diff
@@ -189,6 +190,9 @@
                         domain="[('date_end', '&lt;', context_today().strftime('%Y-%m-%d')), ('recurring_next_date', '=', False)]"
                 />
                 <field name="partner_id"/>
+                <field name="commercial_partner_id"
+                        string="Commerical Entity"
```

Comment:
A spelling issue here. The string attribute could be removed.

Reply:
Good catch